movement/io/load_poses.py

```diff
@@ -91,8 +96,11 @@ def from_numpy(
 
 def from_file(
     file_path: Path | str,
-    source_software: Literal["DeepLabCut", "SLEAP", "LightningPose"],
+    source_software: Literal[
```

Comment:
Pleas also add "Anipose" to this parameter's description in the docstring.

Reply:
done